Change some sample disapprove comments to feel less 'stern'. (#1264)
exercism/website-copy: automated-comments/javascript/general/no_named_export.md

The tests won't pass without an [export](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) called `%{export.name}`.

> You can export constants or functions like this:

Reviewer: ```suggestion
Constants and functions can be exported like this:
```
